src/Coders/Model/Relations/BelongsTo.php

```diff
@@ -12,6 +12,7 @@
 use Illuminate\Support\Fluent;
 use Reliese\Coders\Model\Model;
 use Reliese\Coders\Model\Relation;
+use function Couchbase\defaultDecoder;
```

Comment:
I merged this PR, but I wonder if this line needs to be deleted 😂. I hope I didn't break anything lol.

Reply:
Sorry about that, not sure how that crept in there! Just tested on the 0.0.17 release and everything looks fine. I'll spin up a quick PR to remove that if you want?

Reply:
@CristianLlanos #175